lib/metasploit/framework/compiler/mingw.rb

```diff
@@ -34,6 +34,9 @@ def build_cmd(src)
 
           cmd << "#{self.mingw_bin} "
           cmd << "#{src_file} -I #{INCLUDE_DIR} "
+          if self.include_dirs
+            cmd << " #{self.include_dirs} "
+          end
```

Comment:
```suggestion
         cmd << "#{self.include_dirs.map { |include_dir| "-iquote #{include_dir}" }.join(' ')} " if self.include_dirs.any?
```

Reply:
I'm not sure about the #{include_dir} I think the 's' of `include_dirs` misses